getting-started-tutorial/tutorial-2-Model-training.ipynb

```diff
@@ -0,0 +1,2021 @@
+{
```

Comment:
<p>It may be worth to mention that <code>hub_url</code>  points to the <code>raw</code>  version with <code>{name}</code>  and <code>{tag}</code>  naming convention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>